projects/storefrontlib/src/layout/a11y/keyboard-focus/focus.directive.ts

```diff
@@ -9,7 +9,7 @@ import { KeyboardFocusService } from './services/keyboard-focus.service';
 export class FocusDirective extends LockFocusDirective {
   protected defaultConfig: FocusConfig = {};
 
-  @Input('cxFocus') protected config: FocusConfig = {};
+  @Input('cxFocus') config: FocusConfig = {};
```

Comment:
Lowering visibility on an existing attribute is a breaking change.  What was the intent here?

Reply:
Lowering visibility is a breaking change? I think higher is breaking change :). This is `@input`, so this variable is expected to be updated by others. I don't know why others work, but if I keep it as `protected`, I get build error.

Reply:
No No, it's my bad.  I don't know why, I read the diff the other way around, as if you added `protected`.
Nevermind my comment, all is good with making a @Input public : )